.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "toolchain/esp_nonos_sdk"]
 	path = toolchain/esp_nonos_sdk
-	url = https://github.com/cnlohr/esp_nonos_sdk.git
```

Comment:
@cnlohr I don't know what, if anything, you did to your fork of the SDK, but I pointed it back at espressif's for 3.0